python/lsst/jointcal/utils.py

```diff
@@ -17,13 +17,16 @@
 
 import lsst.log
 import lsst.afw.table
+from lsst.afw.image import fluxFromABMag, abMagFromFlux
 from lsst.afw.geom import arcseconds
 
+MatchDict = collections.namedtuple('MatchDict', ['relative', 'absolute'])
+
 
 class JointcalStatistics(object):
     """Compute statistics on jointcal-processed data, and optionally generate plots."""
```

Comment:
I am uneasy about this design: the class contains the statistics as public attributes, but if I understand the code correctly, these attributes are not computed at construction time. That seems to be a trap for the unwary. Naively I would expect the class to compute the attributes (statistics) at construction, at which point the can examine the statistics and call plotting methods as desired (without altering those statistics).

If a different design is desired, or a redesign is not practical at this time, then please provide ample information here as to how to use this class correctly in this documentation block. E.g. after constructing it, call X to compute the statistics. Is there more than one method for this? After that I assume the user can safely call plotting routines, but if there is more to it than that, please explain.